composites/AlgoliaSearch/SearchBar.js

```diff
@@ -1,47 +1,168 @@
 import React from "react";
 import PropTypes from "prop-types";
+import styled from "styled-components";
+import { intlShape, injectIntl, defineMessages } from "react-intl";
+import debounce from "lodash/debounce";
 
-/**
- * Overrides the default submit action with a custom one.
- *
- * @param {Event} submitEvent The submit event.
- * @param {object} props The React props.
- * @returns {void}
- */
-const onSubmit = ( submitEvent, props ) => {
-	submitEvent.preventDefault();
-	props.submitAction( submitEvent );
+import colors from "../../style-guide/colors.json";
+import SearchIcon from "../../style-guide/svg/search.svg";
+import { Icon } from "../Plugin/Shared/components/Icon";
+
+const messages = defineMessages( {
+	headingText: {
+		id: "searchbar.headingtext",
+		defaultMessage: "Search the Yoast knowledge base",
+	},
+} );
+
+const SearchLabel = styled.label`
+	width: 2em;
+	height: 2em;
+	float: left;
+	margin-top: 0.25em;
+`;
+
+SearchLabel.propTypes = {
+	className: PropTypes.string,
+	htmlFor: PropTypes.string,
```

Comment:
not sure htmlFor should be specified as prop

Reply:
Removed them completely. className also is a react specific property.